optimism.tokenlist.json

```diff
@@ -818,6 +818,50 @@
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
     },
+    {
+      "chainId": 1,
+      "address": "0x0a5e677a6a24b2f1a2bf4f3bffc443231d2fdec8",
+      "name": "dForce USD",
+      "symbol": "USX",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/USX.svg",
+      "extensions": {
+        "optimismBridgeAddress": "0xC5b1EC605738eF73a4EFc562274c1c0b6609cF59"
```

Comment:
Has the implementation address been setup on this proxy?

Reply:
Have verified this proxy contract.